js/contributors.js

```diff
@@ -0,0 +1,24 @@
+
+$(function(){
+  $.ajax({
+    url: "https://api.github.com/repos/fossasia/gci17.fossasia.org/contributors"
+  }).done(function(data){
+    data.forEach(function(contributors){
+      var html = "<div class='col-md-3 col-sm-6'>";
+      html += "<a href="+contributors.html_url+">";
+      html += "<img src='icons/github.png'width='20' height='20'/>"
+      html += "    "+contributors.login;
+      html += "<div class='pic'></a>";
+      html += "<img src="+contributors.avatar_url+"><div class='contribs'><p>";
+      html += contributors.contributions;
+      
+      if (contributors.contributions === 1) {
+        html += " contribution";
+      }
+      else {
```

Comment:
What is the use of this if?

Reply:
For select which needs to add after number of contributions. There are number of contributions under the contributor. If that number is '1' there must be contribution (singular).
ex:- If ABC's number of contributions is 1 there will be like this
       1 contribution (singular)
       If ABC's number of contributions are more than 1 there will be like this
       2 contributions (plural)